packages/geohash/geohash.0.1/opam

```diff
@@ -0,0 +1,41 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "#🌐 geohash converter 🐫 library"
+description: """
+Convert WGS84 lat/lon pairs to [Gustavo Niemeyer](http://niemeyer.net/)s
+[Geohash](http://en.wikipedia.org/wiki/Geohash) and back."""
+maintainer: ["Marcus Rohrmoser <work@mro.name>"]
+authors: ["The GeoHash# programmers"]
+license: "GPL-3.0-only"
+tags: ["Social" "Web" "Geohash" "CGI" "RFC3875"]
+homepage: "https://demo.mro.name/geohash.cgi"
+bug-reports: "https://codeberg.org/mro/geohash/issues"
+depends: [
+  "dune" {>= "3.0"}
+  "optint" {>= "0.3"}
+  "tyre" {>= "0.5"}
+  "ocaml" {>= "4.05"}
+  "odoc" {with-doc}
+]
+build: [
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
```

Comment:
The dune file for the tests should link `geohash_bin.lib` and not `lib` and it should also contain `(package geohash_bin)` otherwise they will not compile on the CI.

ping @mro

Reply:
thanks @mseri, will be back in office on Mon, 12th June and prepare a re-submit.